### PR TITLE
Add more (non-trivial) tests

### PR DESCRIPTION
**Pull request by KellerFuchs**
This tests:
- [x] adding users to groups;
- the queries used by our `libnss_pgsql` configuration:
  - [x] the `getpw{nam,uid}` functions for looking up users;
  - [x] the `getgr{nam,gid}` functions for looking up groups;
  - [ ] the `groups_dyn` and `getgroupmembersbygid` functions that tie together users and groups;
- [x] ~~the query used by Postfix for recipient rewriting~~ (done in #9).

### CODE REVIEW

File: test.sql [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-            "lon": 0
-        },
-        "location": "Undisclosed location",
-        "maxUsers": 1000
+	"inet": ["192.0.2.4"],
+	"coordinates": {
+	    "lat": 0,
+	    "lon": 0
+	},
+	"location": "Undisclosed location",
```
Comment (daurnimator) [inline test.sql]: btw, undisclosed location should just be `NULL`. perhaps make this `"Null Island"` instead?

Comment (KellerFuchs) [inline test.sql]: @daurnimator This isn't from my changes, but I will happily fix it.

**Comment by KellerFuchs** [comment]
Awwwww :3
_@KellerFuchs offers jedihugs._

**Comment by KellerFuchs** [comment]
@daurnimator Would you happen to have more of a clue than I how to write the remaining tests?

**Pull request merged**